packages/core-utils/src/itinerary.ts

```diff
@@ -445,6 +445,69 @@ export function calculatePhysicalActivity(
   };
 }
 
+/**
+ * Sources:
+ * - https://www.itf-oecd.org/sites/default/files/docs/environmental-performance-new-mobility.pdf
+ * - https://www.thrustcarbon.com/insights/how-to-calculate-emissions-from-a-ferry-journey
+ * Other values extrapolated.
+ */
+const CARBON_INTENSITY_DEFAULTS = {
+  walk: 0.026,
+  bicycle: 0.014,
+  car: 0.16,
```

Comment:
Cell AO98 (private car - ICE) shows 0.162 if we want to be precise. Is there a way to distinguish between "normal" driving and ride-hail? The latter is significantly higher (0.276! cell BI98)

Reply:
Sadly no, OTP doesn't differentiate between them in terms of mode.